tests/premade_policy/main.tf

```
@@ -10,16 +10,10 @@ data "terraform_remote_state" "prereq" {
 }
 
 module "premade_policy" {
-  source = "../../"
-
-  providers = {
-    aws = aws
-  }
-
-  create_inspector = true
-  name             = data.terraform_remote_state.prereq.outputs.random_name
-  schedule         = "rate(7 days)"
-  duration         = "180"
-  iam_role_arn     = data.terraform_remote_state.prereq.outputs.iam_role_arn
+  source       = "../../"
```

Please re-insert a blank line below `source`. We try to separate visually the terraform-reserved arguments, from the module-specific arguments.
```suggestion
  source = "../../"

```